pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "ising2D"
+version = "0.1"
+requires-python = ">=3.8"
+
+[project.optional-dependencies]
+test = [
+    "pytest"
```

Comment:
add a comma after "pytest" in accordance with PEP-8 (see the "whitespace-in-expressions-and-statements" section), especially since we'll probably add Ruff at some point